src/UniversalFunctions.jl

```diff
@@ -496,12 +500,211 @@ function psi(uf::Grachev, ζ, tt::HeatTransport)
+        second_exp = -_b_m * _d_m * exp(-_d_m * ζ) * (ζ - _c_m / _d_m)
+
+        return _π_group + ζ * (_a_m + first_exp + second_exp)
+    else
+        # Nishizawa2018 Eq. A1 (L < 0)
+        f_m = f_momentum(uf, ζ)
+        return 1 / f_m
+    end
+end
+
+function phi(uf::Holtslag, ζ, tt::HeatTransport)
+    FT = eltype(uf)
+    if 0 < ζ
+        # Derived from Holtslag1988 Eq. 12
+        _a_h = FT(a_h(uf))
+        _b_h = FT(b_h(uf))
+        _c_h = FT(c_h(uf))
+        _d_h = FT(d_h(uf))
+        _π_group = FT(π_group(uf, tt))
+
+        first_exp = _b_h * exp(-_d_h * ζ)
+        second_exp = -_b_h * _d_h * exp(-_d_h * ζ) * (ζ - _c_h / _d_h)
+
+        return _π_group + ζ * (_a_h + first_exp + second_exp)
+    else
+        # Nishizawa2018 Eq. A2 (L < 0)
+        f_h = f_heat(uf, ζ)
+        return 1 / f_h
+    end
+end
+
+function psi(uf::Holtslag, ζ, tt::MomentumTransport)
+    FT = eltype(uf)
+    if 0 < ζ
+        # Holtslag1988 Eq. 12
+        _a_m = FT(a_m(uf))
+        _b_m = FT(b_m(uf))
+        _c_m = FT(c_m(uf))
+        _d_m = FT(d_m(uf))
+
+        return -_a_m * ζ - _b_m * (ζ - _c_m / _d_m) * exp(-_d_m * ζ) - _b_m * _c_m / _d_m
+    else
+        # Nishizawa2018 Eq. A3 (ζ < 0)
+        f_m = f_momentum(uf, ζ)
+        log_term = log((1 + f_m)^2 * (1 + f_m^2) / 8)
+        return log_term - 2 * atan(f_m) + FT(π) / 2
+    end
+end
+
+function psi(uf::Holtslag, ζ, tt::HeatTransport)
+    FT = eltype(uf)
+    if 0 < ζ
+        # Holtslag1988 Eq. 12
+        _a_h = FT(a_h(uf))
+        _b_h = FT(b_h(uf))
+        _c_h = FT(c_h(uf))
+        _d_h = FT(d_h(uf))
+
+        return -_a_h * ζ - _b_h * (ζ - _c_h / _d_h) * exp(-_d_h * ζ) - _b_h * _c_h / _d_h
+
+    else
+        # Nishizawa2018 Eq. A4 (ζ < 0)
+        f_h = f_heat(uf, ζ)
+        return 2 * log((1 + f_h) / 2)
+    end
+end
+
+function Psi(uf::Holtslag, ζ, tt::MomentumTransport)
+    FT = eltype(uf)
+    _a_m = FT(a_m(uf))
+    _b_m = FT(b_m(uf))
+    _c_m = FT(c_m(uf))
+    _d_m = FT(d_m(uf))
+    if abs(ζ) < eps(FT)
+        if ζ >= 0
+            # Derived from Holtslag1988 Eq. 12
+            exp_term1 = _c_m * ((1 - exp(-_d_m * ζ)) / (ζ * _d_m^2) - 1 / _d_m)
+            exp_term2 = (exp(-_d_m * ζ) - 1) / (ζ * _d_m^2)
+            exp_term3 = exp(-_d_m * ζ) / _d_m
+            return _b_m * (exp_term1 + exp_term2 + exp_term3) - _a_m * ζ / 2
+
+        else
+            # Nishizawa2018 Eq. A13 (ζ < 0)
+            return -FT(15) * ζ / FT(8)
+        end
+    else
+        if ζ < 0
+            # Derived from Holtslag1988 Eq. 12
+            exp_term1 = _c_m * ((1 - exp(-_d_m * ζ)) / (ζ * _d_m^2) - 1 / _d_m)
+            exp_term2 = (exp(-_d_m * ζ) - 1) / (ζ * _d_m^2)
+            exp_term3 = exp(-_d_m * ζ) / _d_m
+            return _b_m * (exp_term1 + exp_term2 + exp_term3) - _a_m * ζ / 2
+        else
+            # Nishizawa2018 Eq. A5 (ζ < 0)
+            return -FT(9) * _a_m * ((_b_m * ζ + FT(1))^(FT(4 / 3)) - 1) / ζ / FT(4) / _b_m^FT(2) - FT(1)
+        end
```

Comment:
✔️

Reply:
As in the other PR and the function below, is there a way to collapse these a bit for the `abs(ζ) < eps(FT) && ζ >= 0` and `abs(ζ) => eps(FT) && ζ < 0` cases?